webapp/action_creators/global_actions.jsx

```diff
@@ -392,7 +392,10 @@ export function newLocalizationSelected(locale) {
     } else {
         Client.getTranslations(
             I18n.getLanguageInfo(locale).url,
-            (data) => {
+            (data, res) => {
+                if (!data && res.text) {
+                    data = JSON.parse(res.text); //eslint-disable-line no-param-reassign
```

Comment:
Maybe change this to the following? There's no point in having ESLint run if we're just disabling it on a line-by-line basis
```
let translations = data;
if (!data && res.text) {
    translations = JSON.parse(res.text);
}
```